packages/engines/src/index.ts

```diff
@@ -62,6 +62,8 @@ path.join(__dirname, '../query-engine-darwin-arm64')
 path.join(__dirname, '../query-engine-debian-openssl-1.0.x')
 path.join(__dirname, '../query-engine-debian-openssl-1.1.x')
 path.join(__dirname, '../query-engine-debian-openssl-3.0.x')
+path.join(__dirname, '../query-engine-linux-static-x64')
+path.join(__dirname, '../query-engine-linux-static-arm64')
 path.join(__dirname, '../query-engine-rhel-openssl-1.0.x')
 path.join(__dirname, '../query-engine-rhel-openssl-1.1.x')
 path.join(__dirname, '../query-engine-rhel-openssl-3.0.x')
```

Comment:
Why is there no binary equivalent to lines 77-81 + 85?

```
path.join(__dirname, '../libquery_engine-linux-arm64-openssl-1.0.x.so.node')
path.join(__dirname, '../libquery_engine-linux-arm64-openssl-1.1.x.so.node')
path.join(__dirname, '../libquery_engine-linux-arm64-openssl-3.0.x.so.node')
path.join(__dirname, '../libquery_engine-linux-musl.so.node')
path.join(__dirname, '../libquery_engine-linux-musl-openssl-3.0.x.so.node')
...
path.join(__dirname, '../query_engine-windows.dll.node')

```

Reply:
Yep, I noticed that too and was going to open a separate PR.

Reply:
#19816